src/style.css

```diff
@@ -0,0 +1,154 @@
+/* #region reset */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+  --main-accent: #060b35;
+  --blue-accent: #00acdc;
+
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+/* #endregion */
+
+/* #region container settings */
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start ;
+  align-items: center;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  font-family: Roboto, sans-serif;
+}
+/* #endregion */
+
+/* #region content */
+.card__photo {
+  padding: 0;
+  height: 134px;
+  width: 160px;
+  margin-bottom: 40px;
+}
+
+.product__name {
+  margin-bottom: 4px;
+
+  font-family: inherit;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: var(--main-accent);
+}
+
+.product__code {
+  line-height: 14px;
+  font-size: 11px;
+  font-weight: 400;
+  font-family: inherit;
+
+  color: #616070;
+}
+/* #endregion */
+
+/* #region rating */
+.rating {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 15px;
+}
+
+.card__stars {
+  display: flex;
+  height: 16px;
+}
+
+.stars__star {
+  background: url(images/star.svg) no-repeat center;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:nth-of-type(-n+4) {
```

Comment:
you need to add styles for all the possible modifiers, so that you could change class in html to stars--2 and number of active stars would change